src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -309,11 +311,30 @@ private async Task WriteServerInfo(ZipArchive archive, JsonOperationContext json
 
             foreach (var route in routes)
             {
-                token.ThrowIfCancellationRequested();
-
+                Exception ex = null;
                 var sw = Stopwatch.StartNew();
-                await InvokeAndWriteToArchive(archive, context, localEndpointClient, route, path, endpointParameters, token);
-                debugInfoDict[route.Path] = sw.Elapsed;
+
+                if (_logger.IsOperationsEnabled)
+                    _logger.Operations($"Starting '{route.Path}'...");
+
+                try
+                {
+                    using (var newToken = CreateOperationToken(TimeSpan.FromMinutes(5)))
+                    using (var mergedToken = CancellationTokenSource.CreateLinkedTokenSource(new[] {newToken.Token, token}))
```

Comment:
FYI, there is a ctor with params there IIRC

Reply:
done